indico/web/client/js/react/components/WTFDateTimeField.jsx

```diff
@@ -70,6 +73,14 @@ export default function WTFDateTimeField({
     [timeField, timeId]
   );
 
+  const filter = (d, meta) => {
+    const dateStr = formatDate(ISO_FORMAT, date);
+    if (disabledDates && disabledDates.includes(dateStr)) {
```

Comment:
```suggestion
    if (disabledDates?.includes(dateStr)) {
```

untested, but that way it should work nice regardless if it's null or an empty array